.github/workflows/update_valid_target.yml

```diff
@@ -0,0 +1,86 @@
+name: Update ckb node assume valid target
+
+on:
+  pull_request:
+    types: [ready_for_review]
+    branches:
+      - master
+
+jobs:
+  ready-for-release:
+    name: Update ckb node assume valid target
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create Branch
+        uses: peterjgrainger/action-create-branch@v2.2.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: 'chore-update-valid-target/${{github.head_ref}}'
+          sha: '${{ github.event.pull_request.head.sha }}'
+
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: 'chore-update-valid-target/${{github.head_ref}}'
+      
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.12.0
+
+      - name: Write env file
+        run: |
+          npm run update:valid-target
+
+      - name: Commit env file
+        uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BASE: ${{ github.head_ref }}
+        with:
+          script: |
+            const fs = require('node:fs')
+            const { BASE, HEAD } = process.env
+            const envFilePath = 'packages/neuron-wallet/.env'
+            const destinationBranch = `chore-update-valid-target/${BASE}`
+            const { data } = await github.rest.repos.getContent({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              path: envFilePath,
+              ref: destinationBranch,
+            })
+            await github.rest.repos.createOrUpdateFileContents({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              path: envFilePath,
+              message: `chore: Update ckb node assume valid target for ${BASE}.`,
+              content: fs.readFileSync(envFilePath).toString('base64'),
+              sha: data.sha,
+              branch: destinationBranch,
+            })
+
+      - name: Create PR
+        uses: actions/github-script@v6
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          BASE: ${{github.head_ref}}
+          HEAD: chore-update-valid-target/${{github.head_ref}}
+        with:
+          script: |
+            const { BASE, HEAD } = process.env
+            const { data: pulls } = await github.rest.pulls.list({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            if (pulls.some(pull => pull.head.ref === HEAD)) {
+              return
+            }
+            github.rest.pulls.create({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              head: HEAD,
+              base: BASE,
+              title: 'chore: Update ckb node assume valid target',
+              body: `This PR uses to update ckb node assume valid target for PR https://github.com/yanguoyu/neuron/pull/${context.issue.number}`,
```

Comment:
The message should refer to the `refs` of this action instead of a hard-coded one